README.md

```diff
@@ -1,27 +1,39 @@
-# Jenkins JNLP slave Docker image
+# Jenkins JNLP Agent Docker image
 
-[`jenkinsci/jnlp-slave`](https://hub.docker.com/r/jenkinsci/jnlp-slave/)
+[![Docker Stars](https://img.shields.io/docker/stars/jenkinsci/jnlp-slave.svg)](https://hub.docker.com/r/jenkinsci/jnlp-slave/)
+[![Docker Pulls](https://img.shields.io/docker/pulls/jenkinsci/jnlp-slave.svg)](https://hub.docker.com/r/jenkinsci/jnlp-slave/)
+[![Docker Automated build](https://img.shields.io/docker/automated/jenkinsci/jnlp-slave.svg)](https://hub.docker.com/r/jenkinsci/jnlp-slave/)
 
-A [Jenkins](https://jenkins-ci.org) slave using JNLP to establish connection.
+This is an image for [Jenkins](https://jenkins.io) agent (FKA "slave") using JNLP to establish connection.
+This agent is powered by the [Jenkins Remoting library](https://github.com/jenkinsci/remoting), which version is being taken from the base [Docker Agent](https://github.com/jenkinsci/docker-slave/) image.
 
 See [Jenkins Distributed builds](https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds) for more info.
 
-Make sure your ECS container agent is [updated](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-update.html) before running. Older versions do not properly handle the entryPoint parameter. See the [entryPoint](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definitions) definition for more information.
+## Running
+
+To run a Docker container
+
+    docker run jenkinsci/jnlp-slave -url http://jenkins-server:port <secret> <agent name>
```

Comment:
Can you wrap up the code using the triple backticks instead of relying on indentation ? 
Much more explicits, and more solid to contributors different indentation settings